src/pages/Transaction.vue

```diff
@@ -142,143 +160,240 @@ export default {
+                    strong {{ ` To : ` }}
+                    <AddressField :address="transfer.to" :truncate="16" copy :name="transfer.to === contract.address && contract.name ?  contract.name : null" />
+                  div(class="col-4")
+                    strong {{ ` Token : ` }}
+                    span {{ transfer.value }}
+                    a(:href="'/address/' + transfer.token.address" style="margin-left: 3px;") {{ transfer.token.symbol }}
+            br(v-if="transfers.length > 0")
+            div(class="fit row wrap justify-start items-start content-start")
+              div(class="col-3")
                 strong {{ `Gas Price Charged: ` }}
-                span {{ getGasChargedGWEI() }} GWEI
-              br
-              div
+              span {{ getGasChargedGWEI() }} GWEI
+            br
+            div(class="fit row wrap justify-start items-start content-start")
+              div(class="col-3")
                 strong {{ `Gas Fee: ` }}
-                span {{ getGasFee() }} TLOS
-              br
-              div
+              span {{ getGasFee() }} TLOS
+            br
+            div(class="fit row wrap justify-start items-start content-start")
+              div(class="col-3")
                 strong {{ `Gas Used: ` }}
-                span {{ trx.gasused }}
-              br
-              div
+              div(class="col-9") {{ trx.gasused }}
+            br
+            div(class="fit row wrap justify-start items-start content-start")
+              div(class="col-3")
                 strong {{ `Gas Limit: ` }}
-                span {{ trx.gas_limit }}
-              br
-              div
+              div(class="col-9") {{ trx.gas_limit }}
+            br
+            div(class="fit row wrap justify-start items-start content-start")
+              div(class="col-3")
                 strong {{ `Nonce: ` }}
-                span {{ trx.nonce }}
-            q-tab-panel( name="details" )
-              div
+              div(class="col-9") {{ trx.nonce }}
+          q-tab-panel( name="details" )
+            div
+              div(class="col-3")
                 strong {{ `Input: ` }}
-                span {{ trx.input_data }}
-              br
-              div
+              div(class="col-9") {{ trx.input_data }}
+            br
+            div
+              div(class="col-3")
                 strong {{ `Output: ` }}
-                span {{ trx.output }}
-            q-tab-panel( name="logs" )
-              .jsonViewer
-                logs-viewer( :logs="getLogs()" )
-            q-tab-panel( name="internal" )
-              InternalTxns( :itxs="trx.itxs" )
+              div(class="col-9") {{ trx.output }}
+          q-tab-panel( name="logs" )
+            .jsonViewer
+              logs-viewer( :logs="getLogs()" :rawLogs="trx.logs" )
+          q-tab-panel( name="internal" )
+            InternalTxns( :itxs="trx.itxs" )
 </template>
 
-<style scoped lang="sass">
+<style lang="sass">
```

Comment:
since these styles aren't namespaced, they need to be at least `scoped`. as written, these styles will "bleed"  

`<style scoped lang="sass">`